### ISSUE TITLE
column-span still not working

### ISSUE THREAD

**Issue by fabiobatalha** [Kozea/WeasyPrint#980]
I did some tests to test the merge-request (https://github.com/Kozea/WeasyPrint/pull/878)

This merge request is related to the issue: #719 

I took the same article displayed in the issue #719 and tried to produce the PDF file.

No changes were noticed on the PDF preview.

The original files are available in the zip below:

[column-span.zip](https://github.com/Kozea/WeasyPrint/files/3780286/column-span.zip)

**Expected result:**

![expected](https://user-images.githubusercontent.com/504142/67709676-cc535480-f994-11e9-875c-66c112c2cce0.png)

**Got:**

![got](https://user-images.githubusercontent.com/504142/67709684-d1b09f00-f994-11e9-8fcf-61316c440e9f.png)

**Comment by Tontyna** [comment]
As the title of #878 announces: It's only a **partial** support.
Of course, `column-span: all` should span across all columns of the **nearest multicol ancestor** but at the moment it looks like it only manages to span across the columns of the **parent** element.

That's why the footnote's title spans across the page, but the figures and tables don't.

BTW: It's your lucky day that WeasyPrint ignores the figures and tables and doesnt crash in the footnote, see #984

**Comment by liZe** [comment]
> Of course, `column-span: all` should span across all columns of the **nearest multicol ancestor** but at the moment it looks like it only manages to span across the columns of the **parent** element.

Exactly. [The code](https://github.com/Kozea/WeasyPrint/blob/a0ccc8d2e6a894d0ae871580a7d4837e088d6582/weasyprint/layout/columns.py#L91-L98) currently looks at the children to find `column-span: all`, but not at their own children.

Changing this behavior is not that easy, because we don't want to check all the grandchildren (nested columns, for example), we probably have to take care about some CSS properties like display and position, and we'll have to split the children using a `resume_at` / `skip_stack` tuple instead of a simple index.
